src/clr-addons/multilingual/abstract-multilingual.ts

```diff
@@ -14,6 +14,7 @@ export abstract class ClrMultilingualAbstract extends ClrAbstractFormComponent {
   @Input('clrSelectedLang') selectedLang: string;
   @Input() readonly: string;
   @Input() maxlength: number;
+  @Input() rows: number;
```

Comment:
pls move this to multilingual-textarea component, as it's not used in single line input

Reply:
done